source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +138,98 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
 * dhcpv6-stateful
 * dhcpv6-stateless
 
-Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn should only accept the 'dhcpv6-stateful' configuration, and fail for the other 2 options. When not provided, this parameter will of course default to the only accepted value.
+Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn will accept either the 'dhcpv6-stateful' or 'dhcpv6-stateless' configuration, while *slaac* is not yet supported. When not provided, this parameter will default to dhcpv6-stateful, since a router is required for the stateless option.
+
+Also, only the **ipv6_address_mode** option will be implemented, since their
+behaviors kind of overlap - e.g. *slaac* and *dhcpv6-stateless* **require** RAs
+to configure their IP addresses, so, no point in having a separate setting to
+configure those.
 
```

Comment:
Is this compatible with neutron? Maybe we should have both with the same value?

Reply:
It **is** compatible, check the subnet validation performed by neutron - follow the following github links:

- https://github.com/openstack/neutron/blob/master/neutron/db/db_base_plugin_v2.py#L656 ( _validate_subnet method, in the pre-commit hook)
- https://github.com/openstack/neutron/blob/master/neutron/db/db_base_plugin_v2.py#L308 ( _validate_ipv6_attributes)
- https://github.com/openstack/neutron/blob/master/neutron/db/db_base_plugin_v2.py#L316

Two checks are done: 
  - one when both attributes are supplied - they have to match 
  - one when only one of them is supplied, and when they are either slaac or dhcpv6_stateless, the subnet's CIDR has to have a /64 prefix.

Also, either of the attributes **require** the **enable_dhcp** subnet option as well, but from my understanding, we're hard-coding it (we encode the subnet in a dhcp_options object, and hard-code it's value in the row2rest mapper method - https://github.com/oVirt/ovirt-provider-ovn/blob/master/provider/neutron/neutron_api_mappers.py#L580)

Reply:
Please "Resolve the conversation" if you consider the explanation to be OK.